app/models/story.rb

```diff
@@ -27,7 +27,7 @@ def estimate_for(user)
     # cache the estimate for each user to not process it every time
     @estimate_for ||= {}
     # use the `estimates` association directly to use the cached query to prevent N+1
-    @estimate_for[user] ||= estimates.select{ |e| e.user == user }.sort_by{|e| e.id}.first
+    @estimate_for[user] ||= estimates.select { |e| e.user == user }.min_by { |e| e.id }
```

Comment:
from this file down, all the changes are just style changes fixed by rubocop, nothing to review